webpack/assets/javascripts/react_app/routes/Audits/constants.js

```diff
@@ -10,5 +10,5 @@ export const AUDITS_PAGE_CLEAR_ERROR = 'AUDITS_PAGE_CLEAR_ERROR';
 
 export const AUDITS_PATH = '/audits';
 export const AUDITS_SEARCH_PROPS = getControllerSearchProps('audits');
-export const AUDITS_MANUAL_URL = version =>
-  `${getManualURL(version)}#4.1.4Auditing`;
+export const AUDITS_MANUAL_URL = () =>
+  getManualURL(`4.1.4Auditing`);
```

Comment:
General javascript question: is there a reason to prefer `'` here given there's no variable interpolation going on?

Reply:
I agree, we can use regular single qoutes
and it doesn't need to be a function since we dropped the version param, right?

Reply:
The function will make the evaluation in runtime (which I think is still desirable).

Reply:
`getManualURL` itself is still dynamic because `getForemanUrl` is still dynamic I think. Not entirely sure, but this must be resolved on the client at runtime, rather than during build time.